tests/TestHelpers/DurableStorage/MemoryStorage.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 
-namespace Tuf\Tests\DurableStorage;
+namespace Tuf\Tests\TestHelpers\DurableStorage;
```

Comment:
Moving this cause a test failure in DurableStorageAccessValidatorTest. I guess our test only run if the PR is against main. I  will fix